src/skia/PathEffect.cpp

```diff
@@ -62,7 +63,11 @@ strokerec
 
         :py:meth:`applyToPath` will return true.
         )docstring")
-    .def("applyToPath", &SkStrokeRec::applyToPath,
+    .def("applyToPath",
+        [] (const SkStrokeRec& strokerec, SkPath* dst, const SkPath& src) {
+            auto dst2 = SkPathBuilder(*dst);
+            return strokerec.applyToPath(&dst2, src);
+        },
```

Comment:
@kyamagu while adjusting this for m140, I realised  that this never worked correctly, right? You cannot update a dst pointer passed in from python and see the result in python afterwards.

This needs to change to return the dst, I think.

Reply:
The change in m140 is that the dst argument has changed type.

Reply:
Yes, this won't work, thanks!